docs/yaml.rst

```diff
@@ -0,0 +1,11 @@
+.. _yaml-head:
+
+YAML Files for Simulation
+*************************
+
+YANK now supports setting up entire simulations with easy to read and write `YAML files <http://www.yaml.org/about.html>`. No longer do you need to remember long command line arguments with many ``--`` flags, you can just write a ``yank.yaml`` file and call ``yank script --yaml=yank.yaml`` to carry out all your preparation and simulations. Below are the valid YAML options. We also have example YANK simulations which use YAML files to run.
+
+.. todo:: This section is a work in progress and not all options are shown yet
+
+.. literalinclude:: ../examples/yank-yaml-cookbook/all-options.yaml
```

Comment:
Maybe it's worth linking also `../examples/yank-yaml-cookbook/combinatorial-experiment.yaml`?